prboom2/src/m_cheat.c

```diff
@@ -444,6 +421,10 @@ static void cheat_clev(char buf[3])
 	  {
 		  epsd = 1;
 	  }
+
+	  // Catch invalid maps.
+	  if (W_CheckNumForName(MAPNAME(epsd, map)) == -1)
+		  return;
```

Comment:
How about a message to doom_printf like "idclev target 36 not found"

Reply:
Yes, nice idea!